src/components/Project/ProjectBasics/ProjectForm/ProjectInfoSection.tsx

```diff
@@ -33,6 +35,12 @@ const ProjectInfoSection: FC<IProjectInfoSectionProps> = ({ project, getFieldPro
       <FormSectionTitle {...getFieldProps('basics')} />
       <div className="form-row">
         <div className="form-col-xl">
+          {projectMode === 'new' && (
+            <TextField
+              {...getFieldProps('name')}
+              rules={{ ...validateMaxLength(200, t), ...validateRequired('description', t) }}
```

Comment:
validateRequired('description', t) is this intentional or should it be validateRequired('name', t)?

Reply:
Fixed to name